### PR TITLE
[#9207] Sort school names for userMapData.json

### LINKED ISSUE #9207: Sort school names for userMapData.json
As per the issues discussed in #9190 , someone needs to go through userMapData.json and confirm the ordering for all lines with the following rule in mind:

When comparing school names A and B, remove all whitespace and symbols (namely, hyphens) from both names, then sort alphabetically.

If we look at an example from the current file, we have "Northwest High School" followed by "Northwestern College". To determine which should actually be first, remove white space and symbols to be left with "NorthwestHighSchool" and "NorthwesternCollege". Afterwards, compare the two strings. Since the 'h' in "High School" comes after the 'e' in "ern", we need to swap the two. 

Repeat this for all lines in the file.

### PR DESCRIPTION
**Pull request by monmanuela**
Fixes #9207 

**PR Checklist**

Ensure that you have:
- [x] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [x] Added/updated tests, if changes in functionality were involved
- [x] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

Following @wkurniawan07 's suggestion [here](https://github.com/TEAMMATES/teammates/issues/9207#issuecomment-433696721), I used javascript's Array.sort() method to reorder the school names for each country.

### CODE REVIEW

**Comment by monmanuela** [comment]
Done!

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -1883,6 +1883,7 @@
       "UNC Pembroke",
       "Union Mine High School",
       "United States Military Academy Preparatory School",
+      "Univeristy of North Georgia",
```
Comment (wkurniawan07) [inline src/main/webapp/data/userMapData.json]: Let's use this opportunity to fix this typo.

Comment (monmanuela) [reply src/main/webapp/data/userMapData.json]: Fixed two typos 👍

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -1883,6 +1883,7 @@
       "UNC Pembroke",
       "Union Mine High School",
       "United States Military Academy Preparatory School",
+      "University of North Georgia",
```
Comment (wkurniawan07) [inline src/main/webapp/data/userMapData.json]: Now this is out of order though...

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -2011,10 +2011,10 @@
       "University of Wisconsin",
       "University of Wisconsin Oshkosh",
       "University of Wisconsin School of Business",
-      "University of Wisconsin, Green Bay",
-      "University of Wisconsin, La Crosse",
       "University of Wisconsin-Madison",
       "University of Wisconsin-Platteville",
+      "University of Wisconsin, Green Bay",
+      "University of Wisconsin, La Crosse",
       "Unniversity of Washington, Seattle",
```
Comment (wkurniawan07) [inline src/main/webapp/data/userMapData.json]: Happened to spot this. Let's fix this as well.

**Comment by monmanuela** [comment]
Done for both. Should I document about the script somewhere?

**Comment by wkurniawan07** [comment]
@monmanuela No need to do anything with the script for now; it's good enough where it is.

**Pull request merged**